pages/Configuring/Uncommon-tips-&-tricks.md

```diff
@@ -114,6 +114,66 @@ bind = $mod, S, movetoworkspace, special:magic
 bind = $mod, S, togglespecialworkspace, magic
 ```
 
+## Show desktop
+
+This approach uses same principle as the `Minimize windows using special workspaces` section.
+It moves all windows from current workspace to a `special` workspace named `desktop`.
+Showing desktop state is remembered per workspace.
+
+Create a script:
+```sh
+#!/usr/bin/env sh
+
+TMP_FILE="/tmp/hyprland-show-desktop"
```

Comment:
Why not per-user in `$XDG_RUNTIME_DIR?`

Reply:
It doesn't really matter, file is deleted right after all windows are restored. But that would solve a case of someone left a workspace on 'desktop shown' mode and another user or same logs in and it doesn't work from first time. Nice catch.